spring-cloud-dataflow-docs/pom.xml

```diff
@@ -92,21 +101,11 @@
 												<include name="**/*.html" />
 											</fileset>
 										</copy>
-										<copy todir="${basedir}/target/contents/reference/htmlsingle">
-											<fileset dir="${basedir}/src/main/docbook">
-												<include name="**/*.css" />
-												<include name="**/*.png" />
-												<include name="**/*.gif" />
-												<include name="**/*.jpg" />
-											</fileset>
+										<copy todir="${basedir}/target/contents/reference/htmlsingle/images">
+											<fileset dir="${basedir}/target/generated-docs/images"/>
```

Comment:
Ah, turns out "There are 2 sets of images - the ones that belong to Asciidoctor (SCDF) and then the ones that belong to the general docs generation … e.g. src/main/docbook/images/ contains the “Note” and “Tip” (Admotion) images, the EPub Cover, and the first page Spring Logo used for the PDF cover."  
```
HTML: “src/main/docbook/images/background.png”
PDF: “src/main/docbook/images/logo.svg” (switched to SVG for sharper rendering)
EPub: “src/main/docbook/images/cover.png”
```